lib/ansible/modules/packaging/os/pkgutil.py

```diff
@@ -64,146 +77,219 @@
     name: CSWnrpe
     site: 'ftp://myinternal.repo/opencsw/kiel'
     state: latest
+
+# Remove a package
+- pkgutil:
+    name: CSWtop
+    state: absent
+
+# Install several packages
+- pkgutil:
+    name: CSWtop,CSWsudo
+    state: present
+
+# Update all packages
+- pkgutil:
+    name: '*'
+    state: latest
+
+# Update all packages and force versions to match latest in catalog
+- pkgutil:
+    name: '*'
+    state: latest
+    force: yes
 '''
 
 import os
 import pipes
 
-def package_installed(module, name):
-    cmd = ['pkginfo']
-    cmd.append('-q')
-    cmd.append(name)
-    rc, out, err = run_command(module, cmd)
-    if rc == 0:
-        return True
-    else:
-        return False
+def packages_not_installed(module, name):
+    # check if each package is installed
```

Comment:
This should be on one line and start with a capital letter.

Reply:
Why not use pythons docstrings to document what the function does?
eg.
```python
def kos_root():
    """Return the pathname of the KOS root directory."""
    global _kos_root
    if _kos_root: return _kos_root
    ...
```